proto/control-api/src/grpc/client.rs

```diff
@@ -0,0 +1,198 @@
+//! gRPC [`ControlApi`] and [`CallbackApi`] clients.
```

Comment:
`ControlApi` and `CallbackApi` implementations are intended to be on different sides. The point of `"client"` feature is to provide everything required for the side which _calls_ Control API, while the `"server"` one for the side which _implements_ Control API.

So, the parts here should be reversed:
- `"client"` feature should enable `ControlApiClient` and `CallbackApiServer`.
- `"server"` feature should enable `ControlApiServer` and `CallbackApiClient`.